data/campaigns/tutorial/scenarios/02_Tutorial_part_2.cfg

```diff
@@ -869,6 +869,24 @@ A full list of abilities and weapons specials, along with traits, may be found i
         [/filter_condition]
 
         {TALK_ABOUT Thrag ( _ "You are close to killing their leader! The unit who finishes him will gain 16 experience points because he is second level. Choose your attacking unit carefully!")}
+
+        [message]
+            speaker=student
+            message= _ "Should I choose a unit that will level up?"
+            message_female= _ "female^Should I choose a unit that will level up?"
+        [/message]
+
+        {TALK_ABOUT Thrag ( _ "That’s a difficult question. Consider which advancements will strengthen your army, and whether the unit should advance now or later.")}
+
+        {TALK_ABOUT Thrag ( _ "Some second level units strengthen your army far more than just having a stronger version of the first level unit. A Captain’s leadership makes every level 1 unit do 25% more damage when he’s next to them. A Druid heals 8 hitpoints instead of 4, and her ability to cure poison is invaluable when you are facing enemies that use it. The powerful, accurate attacks of a Sorceress or Marksman are good for dislodging fortified enemy units. Having these units already levelled on your recall list can make a huge difference in the next battle.")}
+
+        {TALK_ABOUT Thrag ( _ "If your army already has a few units with those abilities, then I’d normally choose a unit that needs 17-24 experience points to advance, so that they’ll be about to level. Units are fully healed when they advance a level, and all units are fully healed between scenarios. Also, until they level, they’ll cost less gold per turn in upkeep.")}
+
+        [message]
+            speaker=narrator
+            caption= _ "This is nearly the end of the tutorial"
+            message= _ "In a campaign you may have ten or even twenty scenarios to build your army, but in this tutorial there’s no-one else to fight after you kill Thrag."
```

Comment:
```suggestion
            message= _ "This tutorial will conclude after you defeat Thrag. However, in a campaign, you may have ten or even twenty scenarios to build your army. Choose your advancements wisely and remember to keep your army versatile enough to adapt to different situations."
```

Reply:
For this one and its caption, I prefer my wordings. The player's train of though should be on advancements at this point, and I want to say "... but on that topic, don't think too hard because your choice doesn't matter in this scenario".

Reply:
How about "but this tutorial will conclude after you defeat Thrag." for the second clause then? If not, the hyphen in "no-one" seems a bit unusual, although this may be a US english vs. UK english thing.

Reply:
I like the "conclude" formulation better than the "there's no one else to fight". The latter feels a bit dispiriting somehow ("We're sorry but no more AI opponents are available at this time") whereas the former is positive ("You're about to win the tutorial").

Besides, carefully planning XP allocation is a good habit to acquire. :)

Reply:
:+1: That could flatten the learning curve for some new players. The `victory` event would be a good place to add that: https://github.com/wesnoth/wesnoth/blob/b789c07e3301e7d8ad2e4f424647e65f29b4dfc7/data/campaigns/tutorial/scenarios/02_Tutorial_part_2.cfg#L1159-L1168

Reply:
:+1: Given the changes in #1222, I think it makes sense to change that line to say "rookie-level", and drop the reference to HttT. Still thinking about what to say about the other two campaigns, how they fit in to the tutorial.

Reply:
What would be best for a new player to play after he finishes the tutorial?

Adding @nemaara who's working on #3648.

Reply:
@nemaara Thanks, maybe stevecotton can use that in his line. I was hoping you would comment on TSG in relation to new players? For example, both AOI and AToTB have hints in their easiest difficulties, and as you're working on TSG, would it be appropriate to add some hints/guidance to TSG?

Reply:
TSG is closer to an intermediate level campaign given its length and unique-ish gameplay in some of the later scenarios. TSG does have its own guidance on the easiest difficulty, but I don't really think it's worth mentioning anything about TSG in the tutorial.

Reply:
@nemaara after #3648, should TSG be considered rookie, novice or intermediate?

I'm thinking about putting a note in the tutorial that the campaign menu sorts by difficulty (by default). However, TSG's _main.cfg still says `rank=10` and `Rookie level`.

Reply:
I guess it could be novice level (?). It was never really something that was thaaaat easy (maybe for experienced players, but not for someone just starting out); I did make it a bit easier, but probably not appreciably so. Its order in the campaign list is still fine, though, it's not as hard as HttT.

Reply:
How does it compare to AOI (which currently rank 15)?

Reply:
I don't know since I haven't played AOI yet. Looking briefly at gold values, it seems like AOI might be harder.

Reply:
Isn't AOI fairly easy on its easiest difficulty? The player usually has a terrain advantage and there aren't fancy objectives/mechanics (AToTB has passwords, and poison without healers). It's been some time since I played these, but I think AOI might be easier than either AToTB or TSG (pre #3648). So maybe AOI should be sorted before AToTB in the campaign screen...

@nemaara The campaign description of TSG says:

> Note: This campaign is designed as an introduction to Wesnoth. The ‘Civilian’ difficulty level is aimed at first-time players.

is that still accurate? Is TSG beatable on Civilian by a first-time player?

Most of the above discussion isn't really about the tutorial. We should probably continue this on elsewhere (on #1222 or #3648 or a new issue) so it's more visible.